src/librustc_trans/trans/context.rs

```diff
@@ -131,6 +128,11 @@ pub struct LocalCrateContext<'tcx> {
     /// Cache of closure wrappers for bare fn's.
     closure_bare_wrapper_cache: RefCell<FnvHashMap<ValueRef, ValueRef>>,
 
+    /// List of globals for static variables which need to be RAUW'ed when
```

Comment:
nit: as far as I can tell the acronym "RAUW" does not occur elsewhere in our code.

So, please expand this occurrence and then include the acronym afterwards in parentheses, effectively defining the acronym.  Like so: "static variables which need to be passed through the LLVM Replace All Uses With (RAUW) functionality"